odoo/addons/base/models/ir_autovacuum.py

```diff
@@ -14,6 +16,10 @@ class AutoVacuum(models.AbstractModel):
     _name = 'ir.autovacuum'
     _description = 'Automatic Vacuum'
 
+    @property
+    def _autovacuum(self):
+        return super()._autovacuum + ('_gc_transient_models', '_gc_user_logs')
```

Comment:
We should move `_gc_transient_models` on... transient models?
Simply define a method `_autovacuum_transient` on TransientModel!
You can even move the whole implementation of transient models on TransientModel.

The method `_gc_user_logs` should be moved to model `res.users.log`.

Reply:
* Moved `_gc_user_logs` to `res.users.log`
* Didn't move `_gc_transient_models` on `model.TransientModel` because it was called for every transient model producing quite a lot of logs. Having just one (isolated) method that loops on every transient to vacuum them is better.